rpcs3/Emu/System.cpp

```diff
@@ -634,7 +634,16 @@ void Emulator::Init()
 	// Load IPC config
 	g_cfg_ipc.load();
 	sys_log.notice("Using IPC config:\n%s", g_cfg_ipc.to_string());
-	g_fxo->get<IPC_socket::IPC_server_manager>().set_server_enabled(g_cfg_ipc.get_server_enabled());
+
+	if (g_cfg_ipc.get_server_enabled())
+	{
+		g_fxo->need<IPC_socket::IPC_server_manager>();
+	}
+
+	if (auto server = g_fxo->try_get<IPC_socket::IPC_server_manager>())
+	{
+		server->set_server_enabled(g_cfg_ipc.get_server_enabled());
```

Comment:
I guess this was flawed to begin with, read the config value in IPC_server_manager constructor.

Reply:
I changed it to only create the entire object if configured.